python/ray/_private/log_monitor.py

```diff
@@ -386,13 +387,19 @@ def flush():
                     raise
 
             if file_info.file_position == 0:
-                if "/raylet" in file_info.filename:
+
+                def is_logfile(key, file_info):
```

Comment:
Is it possible to implement this using https://docs.python.org/3/library/pathlib.html?

Reply:
`os.path.split()` works on both `/` and `\` on windows. There is no such functionality in `pathlib`

Reply:
ahh, but there is `pathlib.Path(file_info.filename).as_posix()`